### ISSUE TITLE
Reduce noise

### ISSUE THREAD

**Issue by ebfull** [LeastAuthority/leastbot#27]
The bot spits out messages that aren't terribly useful (new comments, labelled issues). It might dissuade participation in our channel.

**Comment by nathan-at-least** [comment]
Here's a feature set I imagine would help with noise:
- for label changes, only make an announcment if it's in a whitelist of important labels (at least "review needed").  (Also, of course, announce what the actual label is!)
- announce PR closes.  (Annoying with our workflow when we have "superceded" PRs.)

I don't think we need to announce anything else really.  _Maybe_ we could have an intermittent annoucement of all of the changes every 30 minutes or something, that would be like: "The following tickets were updated: ...; The following wiki pages were edited: ...."  Let's leave this idea for a future ticket.

**Comment by nathan-at-least** [comment]
Oh, instead of PR closes, we could announce PR _merges_.  That should be a work-around for our annoying "superceded" workflow.

**Comment by zancas** [comment]
I'm not sure what the policy is that generates "superceded" PR closes, but github pull requests are defined between branches...  so one can always push new commits to the "from" branch that is associated with a pull request, and what is being considered for merge will automatically update.

Then you can just add comments in the PR to accompany (reference) the new commit.

In My Humble Opinion it's usually best to just extend an existing pull request...   maybe it's necessary to supercede in some cases...  but those should be pretty rare, and therefore not generate much noise.

**Comment by ebfull** [comment]
> In My Humble Opinion it's usually best to just extend an existing pull request...

I agree in general. However, we need to be careful about removing commits or rewriting history. Force pushes are generally frowned upon in security-sensitive projects on github. If we have to rebase, it should be a new pull request. Auditors will enjoy the extra context and information.

Part of the reason for the frequent "superseded" PR closures is being addressed by https://github.com/Electric-Coin-Company/zerocashd/issues/304.
